### PR TITLE
Add initial 2023 Community Update

### PR DESCRIPTION
**Pull request by ericholscher**
This is mostly just covering conferences and the Community Board updates.
Happy for suggestions of other content,
if folks have specific things to mention.

----
:books: Documentation preview :books:: https://writethedocs-www--2004.org.readthedocs.build/

### CODE REVIEW

File: docs/blog/2023-community-update.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+and explore our new venue in Portland.
+The `photos <https://www.flickr.com/photos/writethedocs/albums/72177720308088427>`_ and `videos <https://www.youtube.com/playlist?list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2>`_ from the conference are available if you haven't caught up with them yet.
+
+2024 Conferences
+----------------
+
+We are still in the planning phrase for our 2024 conferences.
+We expect Portland to return in-person,
+and we will make a decision about the other conferences after the 2023 editions happen later this year.
+Overall, we are trying to balance between the accessibility of virtual events and the human interaction value of in-person events.
```
Comment (thatdocslady) [inline docs/blog/2023-community-update.rst:31]:
```suggestion

Overall, we are trying to balance between the accessibility of virtual events and the human interaction value of in-person events.
```

Comment (thatdocslady) [reply docs/blog/2023-community-update.rst:31]: Adding paragraph break for scannability.

File: docs/blog/2023-community-update.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+given that running a conference is a lot of work across many members of our organizer teams.
+
+We're really grateful for the support that the community has given us throughout the past few years,
+and we're excited to see what the future holds for our events.
+
+Community Board Updates
+-----------------------
+
+Our Community Board has been meeting each quarter for all of 2023,
+and we have been posting `meeting notes <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ for each meeting.
```
Comment (thatdocslady) [inline docs/blog/2023-community-update.rst]:
```suggestion
and we have been posting `meeting summaries <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ for each meeting.
```

File: docs/blog/2023-community-update.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+The current structure of the board is to have each team lead from one of our teams, and each of our conference chairs, as a member.
```
Comment (thatdocslady) [inline docs/blog/2023-community-update.rst]:
```suggestion
The current structure of the board includes the team leads from each of our teams, and each of our conference chairs.
```

**Review by thatdocslady — COMMENTED** [review]
Looks great, a few nits :)

File: docs/blog/2023-community-update.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
+.. post:: Aug 14, 2023
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2023
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here filling in for the Newsletter team as they take their normal August break.
```
Comment (plaindocs) [inline docs/blog/2023-community-update.rst:10]: Filling in implies to me that you're doing a newsletter. How about 

```suggestion
Eric Holscher here, with a WTD community update while the Newsletter team take their normal August break.
```

File: docs/blog/2023-community-update.rst (hunk 1 above)
Comment (plaindocs) [inline docs/blog/2023-community-update.rst:31]:
```suggestion
Overall, we are trying to balance the accessibility of virtual events with increased human interaction of in-person events.
```

File: docs/blog/2023-community-update.rst [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+2024 Conferences
+----------------
+
+We are still in the planning phrase for our 2024 conferences.
+We expect Portland to return in-person,
+and we will make a decision about the other conferences after the 2023 editions happen later this year.
+Overall, we are trying to balance between the accessibility of virtual events and the human interaction value of in-person events.
+The other major factor is organizer energy and availability,
+given that running a conference is a lot of work across many members of our organizer teams.
```
Comment (plaindocs) [inline docs/blog/2023-community-update.rst:33]:
```suggestion
running a conference is a lot of work across many members of our organizer teams.
```

File: docs/blog/2023-community-update.rst [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+We're really grateful for the support that the community has given us throughout the past few years,
+and we're excited to see what the future holds for our events.
```
Comment (plaindocs) [inline docs/blog/2023-community-update.rst:36]:
```suggestion
We're really grateful for the support that the community has given us throughout the past few difficult years, and we're excited to see what the future holds for our events while continuing to find solutions that work for us and our community.
```

might need increased rewording? @thatdocslady

**Review by plaindocs — APPROVED** [review]
Gah, I didn't submit this. Safe to ignore I think.

**Pull request merged**